applications/areg-portal/frontend/src/theme/Theme.tsx

```diff
@@ -149,6 +155,23 @@ const theme = createTheme({
         },
       },
     },
+    MuiMenu: {
+      styleOverrides: {
+        paper: {
+          width: "18.875rem",
```

Comment:
This width is better removed: any container adapts to its content by default

Reply:
I was trying to match Figma design, where the dropdown menu is wider than the content. Is ok if it's narrower than the mock up?

Reply:
we should try to match the sense of spacing that's in the mockup, but no need to match the size. Also being this in the theme, every menu in the size would be expected to have the same size, which is not ideal in a real setup

Reply:
done